File: A1/Java/A1.java
import java.util.*;

public class A1{

	public static void main(String[] args) {
		Scanner input = new Scanner(System.in);

		System.out.println("Enter your test string");
		String s1 = input.nextLine();


		System.out.println("\nEnter another string and I'll tell you if it's inside your first string");
		String s2 = input.next();

		if (s1.contains(s2)) {
			System.out.println("FOUND");
		}
		else {
			System.out.println("NOT FOUND");
		}
	}

}

File: A1/C++/A1.cpp
#include <iostream>
#include <string>
using namespace std;

int main(){
	string s1;
	string s2;

	cout << "Enter your first string" << endl;
	getline(cin, s1);

	cout << "Enter your second string" << endl;
	cin >> s2;

	cout << "s1: " << s1 << endl;
	cout << "s2: " << s2 << endl;

	int found = s1.find(s2);


	if (found != -1)
		cout << "FOUND" << endl;
	else
		cout << "NOT FOUND" << endl;


return 0;
}
